### PR TITLE
Камелина - Лабораторная работа #2

### PR DESCRIPTION
**Pull request by JuliaKamelina**
@AnastasiaaSenina @UsovaMA please review

### CODE REVIEW

File: code/float_vector_metrics/viewmodel/test_viewmodel.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import unittest
+
+from float_vector_metrics.viewmodel.viewmodel import VectorMetricsViewModel
+
+
+class TestVectorMetricsViewModel(unittest.TestCase):
+    def setUp(self):
```
Comment (UsovaMA) [inline code/float_vector_metrics/viewmodel/test_viewmodel.py:7]: Start a test title with "test_"

File: code/float_vector_metrics/view/view.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
+import tkinter as tk
+from tkinter import ttk
+
+from float_vector_metrics.viewmodel.viewmodel import VectorMetricsViewModel
+
+
+class GUI(ttk.Frame):
+    default_sticky = tk.W + tk.E + tk.N + tk.S
```
Comment (UsovaMA) [inline code/float_vector_metrics/view/view.py:8]: Unused variable

Comment (JuliaKamelina) [reply code/float_vector_metrics/view/view.py]: Done

**Comment by UsovaMA** [comment]
:+1:

**Review by UsovaMA — APPROVED** [review]

**Review by AnastasiaaSenina — CHANGES_REQUESTED** [review]
If entered vectors is not correct, for example, [1, a, 2] and [1, 2, 3], the metrics were calculated ( a number was received, but an error message was expected). Please fix it.

**Comment by JuliaKamelina** [comment]
@AnastasiaaSenina Done

**Review by AnastasiaaSenina — APPROVED** [review]
👍

**Pull request merged**